karabo_bridge/client.py

```diff
@@ -66,10 +69,15 @@ def __init__(self, endpoint, sock='REQ', ser='msgpack'):
         else:
             raise NotImplementedError('socket is not supported:', str(sock))
 
+        if timeout is not None:
+            self._socket.setsockopt(zmq.RCVTIMEO, timeout)
+        self._recv_ready = False
+
         self._pattern = self._socket.TYPE
 
         if ser == 'msgpack':
-            self._deserializer = partial(msgpack.loads, raw=False)
+            self._deserializer = partial(msgpack.loads, raw=False,
+                                         max_bin_len=0x7fffffff)
```

Comment:
I changed back the maximum length for binary type in the unpacker. Other types should not break with the new limits I think

Reply:
Did the default `max_bin_len` break the previous test?

Reply:
The previous default was `0x7fffffff` which correspond to 2 GB. And now they changed it to 1MB I think.
The test that breaks is the `version1` where the `ndarray`s are stored as binary strings in the msgpack buffer, and can be very big.
In principle the version 1 of the protocol is not used anymore and is only here for historical reasons. We could also think of removing it at some point and the new defaults values of msgpack should not be a problem.

Reply:
Thank you for the explanation! 

I agree that we can decide to deprecate something in one release and then remove it completely afterwards. I don't think anyone would use the version 1.

LGTM